src/DependencyInjection/DI/test/Microsoft.Extensions.DependencyInjection.Tests.csproj

```diff
@@ -6,6 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="Newtonsoft.Json" />
```

Comment:
?

Reply:
I use it in tests to make sure JSON is valid and reformat it before asserting.